config/locales/es.yml

```diff
@@ -6,3 +6,5 @@ es:
   book_publisher: "Editorial"
   rent_start_date: "Desde"
   rent_end_date: "Hasta"
+  subject_expire_rent_email: "Tu reservación ha expirado."
+  
```

Comment:
please remove this new line, here is innecesary

Reply:
Done